src/pychart/_interpreter/statement.py

```diff
@@ -81,3 +81,34 @@ def __call__(self, environment: Environment):
             self.if_body(environment)
         elif self.else_body and not test_result:
             self.else_body(environment)
+
+
+class pychBreakStatementExceptionJeffer(Exception):
+    pass
+
+class Break(Stmt):
+    def __call__(self, environment: Environment):
+        raise pychBreakStatementExceptionJeffer("The jeeffer did not correspond")
+
+
+class While(Stmt):
+    
+    test: Expr
+    body: Stmt
+
+    def __init__(
+        self,
+        test: Expr,
+        body: Stmt,
+    ):
+        self.test = test
+        self.body = body
+
+    def __call__(self, environment: Environment):
+        while self.test(environment):
+            try:
+                self.body(environment)
+            except pychBreakStatementExceptionJeffer:
```

Comment:
See previous comment
```suggestion
            except BreakStatementException:
```